EDI/Operation/01_ManageDefinitions.md

```diff
@@ -0,0 +1,249 @@
+
+    ![Tabs added](../../Assets/Screenshots/EDI/Operation/ExportDefinitionCreateAfterSelection.png "[Tabs added]")
+
+4. Click the *Default connection* drop-down list and select the connection to the related business partner. 
+
+5. Enter the access key of your connection in the *Access key* field.
+     The access key is a key that you can give to your business partner to access your *Actindo* tenant without logging in, for example, to return confirmation messages.<!---Stimmt das?--> You can freely specify any string.  
+
+     If you are currently creating the definition for the line items, continue with [Get line item segments](./04_ManageLineItemsFileStt.md#get-line-item-segments).
+
+6. If you are in the header definition, click the *File settings* tab.  
+    The *File Settings* tab is displayed.
+
+    ![File settings tab](../../Assets/Screenshots/EDI/Operation/ExportDefEmptyFileSettings.png "[File settings tab]")
+
+7. Click the *Release* drop-down list and select the EDIFACT version to which you want to refer. Information on the relevant version is available in the implementation guide of your business partner.
+
+8. Click the *Message type* drop-down list and select the message type which you want to set up, for example INVOIC.
+
+9. Click the [SAVE] button.   
+    - The *Submitting data* screen is displayed. After that, the *Exporter definitions* view is displayed again, the newly created definition is added. 
+    - An attribute set has been created automatically and has been added to the attribute sets of the *DataHub* module. 
+    - If you now open the definition again for editing, both the *Data Sources* tab and the *Mapping settings* tab will be displayed in addition. 
+
+
+## Edit basic definition settings 
+
+Edit basic data for the export definition. You can edit a definition setting for the header as well as for the line items of a message.
+
+
+#### Prerequisites
+
+- At least one connection has been created, see [Create connection](../Integration/01_ManageConnections.md#create-connection).
+- At least one definition setting has been created, see [Create basic definition settings](#create-basic-definition-settings).
+
+#### Procedure
+
+*DataHub Exporter > Exports*
+
+![Export definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Export definitions]")
+
+1. Click the definition whose settings you want to edit in the list of export definitions.   
+    The *Edit definition "definition name"* view is displayed. The *Settings* tab is displayed by default.
+
+    ![Edit definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionEdit.png "[Edit definition]")
+
+2. Edit the data you want to change.
+
+3. Click the [SAVE] button.   
+    The *Submitting data* screen is displayed. After that, the *Exporter definitions* view is displayed again. 
+
+
+
+## Prepare definition of file settings
```

Comment:
Diese Info besser jeweils in File Settings und Data source integrieren? Hier in Definitions finde ich verwirrend.

Reply:
Es sind zwei verschiedene Herangehensweisen. Diese ist hier wohl die wahrscheinlichere